### ISSUE TITLE
Login problem using RForcecom Sanbox Environment

### ISSUE THREAD

**Issue by jugadupa** [hiratake55/RForcecom#36]
I tried to connect well:
instanceURL <- "https://test.salesforce.com/"
also
instanceURL <- "https://cs26.salesforce.com/"
too
instanceURL <- "https://nn--test.cs26.my.salesforce.com/"

But it has not been possible, in the history login salesforce is no record of the call. You can connect to a sandbox environment for Rforce?

Regards 
Juan Gabriel

**Comment by ReportMort** [comment]
@jugadupa Yes, you can connect to a testing environment via RForcecom. It will run exactly like connecting to a live environment. You might want to double check your password and security token match the username for your sandbox. It might be different from your live environment. Here is an example of connecting to a test environment from the package:

```
rforcecom.login(username='{my-username}', 
                password='{my-password}{my-security-token}', 
                loginURL='https://test.salesforce.com/')
```

**Comment by TonyWhiteSMS** [comment]
Hello @ReportMort 
I am having this issue as well, currently the version is listed as 0.8, and when I try the above I get this error
unused argument (loginURL = "https://test.salesforce.com/")

Looking at http://rforcecom.plavox.info/ it lists 0.8 as the latest.

Thanks
Tony

**Comment by TonyWhiteSMS** [comment]
I have built from the GIT code, used these instructions http://kbroman.org/pkg_primer/pages/build.html

**Comment by abeburnett** [comment]
You actually don't need to build it manually. You can just use devtools to
download and install the current version from git which should resolve your
issues. Once you install devtools then you just do install_github...

Install devtools and use directions:

https://github.com/hadley/devtools/blob/master/README.md

Easy peasy!

**Comment by ReportMort** [comment]
Yeah, the install should be a one-liner. Below are 2 examples on how to install from Github:
```
# install from default branch
devtools::install_github('hiratake55/RForcecom')
# install from a particular branch
devtools::install_github('ReportMort/RForcecom', ref='metadata-api')
```

@TonyWhiteSMS as far as your original question about logging in, if you just run the command `rforcecom.login` without any arguments or parenthesis, then it will show you the function definition. You might want to use this double check that whatever version of RForcecom you've got that it is showing loginURL as a usable parameter.
